test/util/factories/profile_factory.dart

```diff
@@ -32,7 +32,12 @@ class ProfileFactory extends ModelFactory<Profile> {
       username: username ?? faker.internet.userName(),
       email: email ?? faker.internet.email(),
       description: getNullableParameterOr(description, faker.lorem.sentences(random.nextInt(2) + 1).join(' ')),
-      birthDate: getNullableParameterOr(birthDate, faker.date.dateTime()),
+      birthDate: getNullableParameterOr(
+          birthDate,
+          faker.date.dateTime(
```

Comment:
just controlling the year is not enough, this could still result in wrong ages. It would be better to use `dateTimeBetween`.